### PR TITLE
Use definition list for user info

### LINKED ISSUE #108: Use description list for descriptions
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

This includes:

- User info
- project info

One may style it to look like table (See mastodon profile table for example)

### PR DESCRIPTION
**Pull request by DDHLong**
resolved #108

### CODE REVIEW

File: src/acanban/static/style.css [hunk 1]
```diff
@@ -148,7 +172,12 @@ textarea {
 }
 
 /* specialized containers */
-.block { padding-bottom: 1.5em }
+.block { 
+    margin: 1.5rem;
+    padding: 1.5em;
+    box-shadow: 0 8px 32px 0 rgba( 31, 38, 135, 0.37 );
```
Comment (McSinyx) [inline src/acanban/static/style.css:178]: Please use relative sizes, also for the border thickness.

File: src/acanban/templates/user.html [hunk 2]
```diff
@@ -22,16 +22,22 @@
...
-  <div><b>Student ID:</b> {{ user['student-id'] }}</div>
-  {% endif %}
-  {% if user.department %}
-  <div><b>Department:</b> {{ user.department }}</div>
-  {% endif %}
-
+  <dl>
+    <dt><b>Role</b></dt>
+    <dd>{{ user.role }}</dd>
+    <dt><b>Username </b></dt>
```
Comment (McSinyx) [inline src/acanban/templates/user.html:28]: I suppose we can set font weight here to even make it easier to read the template.  Also there's a unnecessary space after *Username*.

File: src/acanban/templates/projects.html [hunk 3]
```diff
@@ -4,12 +4,12 @@
...
   {% for project in projects %}
   <div class=block>
-    <h2><a href=/p/{{ project.id }}>{{ project.name }}</a></h2>
+    <h2><a href=/p/{{ project.id }}><b>Title:</b> {{ project.name }}</a></h2>
     <p><b>Supervisors:</b> {{ project.supervisors | map('userlink')
                               | join(', ') | safe }}</p>
     <p><b>Students:</b> {{ project.students | map('userlink')
                            | join(', ') | safe }}</p>
-    <p>{{ project.description }}</p>
+    <p><b>Description:</b> {{ project.description }}</p>
```
Comment (McSinyx) [inline src/acanban/templates/projects.html:12]: I think the colon can be dropped now.

**Review by McSinyx — COMMENTED** [review]
I saw the screenshot from the chatroom and suppose it is for this.  Next time please also attach it here for the ease of review :wink:

**Review by Huy-Ngo — APPROVED** [review]
Besides Phong's change requests I think it's fine.

File: src/acanban/templates/user.html (hunk 2 above)
Comment (McSinyx) [reply src/acanban/templates/user.html:28]: Sorry if I wasn't clear, I meant to drop the `b` tags here.

**Review by McSinyx — APPROVED** [review]
The diff is looking good, although I recommend you manually running through it before merging (I'm preparing dinner ATM :smile:).

**Pull request merged**